experimental/GModule/GModule.jl

```diff
@@ -569,7 +570,7 @@ module RepPc
 using Oscar
 
 Base.pairs(M::MatElem) = Base.pairs(IndexCartesian(), M)
-Base.pairs(::IndexCartesian, M::MatElem) = Base.Pairs(M, CartesianIndices(axes(M)))
+Base.pairs(::IndexCartesian, M::MatElem) = Base.Iterators.Pairs(M, CartesianIndices(axes(M)))
```

Comment:
Why? as far as I can see, they are identical Base.Pairs and Base.Iterators.Pairs